src/App.js

```diff
@@ -36,29 +36,35 @@ class App extends Component {
     }
     return (
       <section>
-        <header>
-          <Link to='/'>
-          <h1><img className='logo' src={Tomatillos}/>Rancid Tomatillos</h1>
-          </Link>
-          <h2> Welcome {this.state.user.name || 'Movie Goer'}!</h2>
-            <Link to='/signin'>
-            <button type="button">Login</button>
+      <BrowserRouter>
+        <Route>
+          <header>
+            <Link to='/'>
+            <h1><img className='logo' src={Tomatillos}/>Rancid Tomatillos</h1>
             </Link>
-          <button
-            type="button"
-            onClick={event => this.submitLogout(event)}>Logout</button>
-          <img className='flim-reel' src={Film}/>
-          <Login setUser={this.setUser} />
-        </header>
-        <Switch>
-          <Route exact path='/signin' render={() => <Login setUser={this.setUser} /> } />
-          <Route path='/' Component = <Movies /> />
-          // <Route path='/' render={() => <Movies movies={this.state.movies} /> } />
-          // <Route path='/logout' render={() => <App />} />
-        </Switch>
-        // <section className='all-cards'>
-        //   <Movies movies={this.state.movies} />
-        // </section>
+            <h2> Welcome {this.state.user.name || 'Movie Goer'}!</h2>
+              <Link to='/signin'>
+              <button type="button">Login</button>
+              </Link>
+            <button
+              type="button"
+              onClick={event => this.submitLogout(event)}>Logout</button>
+            <img className='flim-reel' src={Film}/>
+            <Login setUser={this.setUser} />
```

Comment:
Is the Login always being rendered? It seems like when you navigate to the `/signin` url you would see two `Login` components.